.github/workflows/tests.yml

```diff
@@ -139,7 +139,9 @@ jobs:
       - name: Create pull request branch
         run: git switch -c "bump-changelog-for-${RELEASE_VERSION}"
       - name: Update changelog
-        run: pipx run towncrier build --yes --version $RELEASE_VERSION
+        run: |
```

Comment:
I would prefer this to be a nox target invocation, in case Github is down, I still want to perform this work locally.

Reply:
Done :+1: See it running in my fork here: https://github.com/Gitznik/pipx/actions/runs/7546614725/job/20544738799